### PR TITLE
RFC-1: author response

### PR DESCRIPTION
**Pull request by joshmoore**
Just *barely* in time to have met my summer goal of replying to the reviews of [RFC-1](https://ngff.openmicroscopy.org/rfc/1), I'm opening this update to the RFC process proposal. There is likely some need for clarifications before the changes are merged and sent out for re-review (R1). I would ask interested members of the community two favors:

### A. Endorsements (easier)

Since the first version of RFC-1 in #222 did not exist in order to request endorsements, I did not request them. Now that RFC-2 and RFC-3 along with #253 and #254 (a.k.a. RFC-4 and RFC-5) have successfully demonstrated the joy of endorsing RFCs, **I would ask anyone who supports RFC-1 to add an emoji below. (Instructions provided)**

### B. Clarifications (bit more work)

Additionally, if you would like to give a careful read of the document and identify anything that is confusing, illogical, or otherwise to be avoided, I'd appreciate the additional eyes. You are additionally welcome to submit an additional, long-form Comment. (Examples are in the built webpages: https://ngff--258.org.readthedocs.build/rfc/1/comments/)

Reviewers are of course welcome to be involved in the clarifications, but need not do so. 

----

⚠️ Emojis on this PR will be considered for endorsements:
 - 👍🏽 ❤️ 🚀 😄 🎉 -- you will be listed on the RFC as an endorser.
 - 👎🏽 ☹️ -- I will contact you regarding how you would like to express your objection.
 - 👀: no special meaning

### CODE REVIEW

File: rfc/1/index.md [hunk 1]
```diff
@@ -1,79 +1,134 @@
...
 it was expected that all parties agree before a specification change was
-considered. This made it especially difficult both for **Authors** as well as
-**Reviewers** to know when a suggested change was adopted since PR comments
-could re-open a discussion. It also left a significant burden on **Editors** to
+considered. This made it especially difficult both for those proposing as well as those reviewing changes to know when a suggested change was adopted since PR comments
+could re-open a discussion. It also left a significant burden on maintainers to
 draw a line where further discussion was not possible. Such a situation leads
 to slower specification evolution and potential deadlocks.
 
+Additionally, extended GitHub discussions had become difficult to follow. Without some curation of the content, it was unclear to the those not deeply involved from the beginning to see what decisions had been made, by whom, when and why. There were also limited options for managing the often complex conversions and differing opinions that arise in technical conversations.
```
Comment (will-moore) [inline rfc/1/index.md:86]: extra "the" in `unclear to the those..`

Comment (joshmoore) [reply rfc/1/index.md]: Thanks. (If you use the +/- button 

image

on any minor changes, I can adopt them all in one fell swoop)

**Review by will-moore — COMMENTED** [review]
Picked up a couple of typos...

**Comment by joshmoore** [comment]
Thanks, all, for the endorsements. One last call for endorsements or other clarifications before I send this out for re-review.

**Pull request merged**